src/shared/static/static.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Inject, Logger, Param, Res } from '@nestjs/common';
+import { Response } from 'express';
+import { ApiOkResponse, ApiTags } from '@nestjs/swagger';
+import { STATIC_SERVICE, StaticService } from './static-service.interface';
+
+@ApiTags('images')
+@Controller('images')
+export class StaticController {
+  private readonly logger: Logger = new Logger(StaticController.name);
+
+  constructor(@Inject(STATIC_SERVICE) private readonly staticService: StaticService) {}
+
+  @Get('/:fileName')
+  @ApiOkResponse()
+  async downloadPngAndRedirect(@Param('fileName') fileName: string, @Res() res: Response) {
```

Comment:
not sure why we need controller for this purpose 
could frontend get the image by direct url?

Reply:
something like this 
https://stackoverflow.com/questions/73302353/using-getobject-from-aws-to-display-an-image-in-the-browser